### PR TITLE
Move MinGW test to allow_failures

### PR DESCRIPTION
**Pull request by isuruf**
This test fails on master as well, but the test succeeded when the last PR was merged on master

### CODE REVIEW

**Comment by certik** [comment]
Does it still get tested? It seems appveyor simply skips this test now.

I thought we have figured out the cause of the failure, didn't we?

**Comment by isuruf** [comment]
> Does it still get tested? It seems appveyor simply skips this test now.

I'll try to fix that.

>I thought we have figured out the cause of the failure, didn't we?

That was a different failure.

File: appveyor.yml [hunk 1]
```diff
@@ -41,6 +41,13 @@ environment:
       COMPILER: MinGW-w64
       PLATFORM: x64
 
+matrix:
```
Comment (srajangarg) [inline appveyor.yml]: I tried this. I think it just doesn't  run this job, as @certik pointed out.

**Comment by srajangarg** [comment]
Closing in favour of #1033

**Pull request closed without merging**